src/main/java/net/neoforged/neoforge/fluids/SimpleFluidContent.java

```diff
@@ -40,6 +47,67 @@ public boolean isEmpty() {
         return this.fluidStack.isEmpty();
     }
 
+    public Fluid getFluid() {
+        return fluidStack.getFluid();
+    }
+
+    public Holder<Fluid> getFluidHolder() {
+        return fluidStack.getFluidHolder();
+    }
+
+    public boolean is(TagKey<Fluid> tag) {
+        return fluidStack.is(tag);
+    }
+
+    public boolean is(Fluid fluid) {
+        return fluidStack.is(fluid);
+    }
+
+    public boolean is(Predicate<Holder<Fluid>> predicate) {
+        return fluidStack.is(predicate);
+    }
+
+    public boolean is(Holder<Fluid> holder) {
+        return fluidStack.is(holder);
+    }
+
+    public boolean is(HolderSet<Fluid> holders) {
+        return fluidStack.is(holders);
+    }
+
+    public int getAmount() {
+        return fluidStack.getAmount();
+    }
+
+    public FluidType getFluidType() {
+        return fluidStack.getFluidType();
+    }
+
+    public boolean is(FluidType fluidType) {
+        return fluidStack.is(fluidType);
+    }
+
+    public boolean matches(FluidStack other) {
+        return FluidStack.matches(fluidStack, other);
+    }
+
+    public boolean isSameFluid(FluidStack other) {
+        return FluidStack.isSameFluid(fluidStack, other);
+    }
+
+    public boolean isSameFluidSameComponents(FluidStack other) {
+        return FluidStack.isSameFluidSameComponents(fluidStack, other);
+    }
+
+    public boolean is(SimpleFluidContent content) {
```

Comment:
should probably be `isSameFluidSameComponents`

Reply:
Renamed 👍 was originally matching the `is(Fluid|FluidType|Holder|HolderSet|TagKey)` methods but since its matching the `FluidStack` and components the rename does make sense